docs/source/overview.rst

```diff
@@ -1,9 +1,9 @@
 Overview
 ========
 
-openprocurement.auctions.appraisal contains documentaion concerning the type of Small Scale Privatization auctions.
+openprocurement.auctions.appraisal іncludes auction insider documentation.
 
-The procedure to be used is **sellout.insider** which states for privatization's objects.
+The procedure to be used is **appraisal.insider**.
 
 Features
 --------
```

Comment:
Organizers can't edit procedure's significant properties (*Auction.value*, etc.) - no, they can (and all the fields are to be mentioned in api-docs)

 The `minimalStep` field is optional (value that will be always automatically set is 0). - to be removed